### ISSUE TITLE
A quick demo would be useful

### ISSUE THREAD

**Issue by 8ctopus** [NoiseByNorthwest/php-spx#211]
For new people discovering SPX, taking it for a test drive is quite cumbersome (besides the link the analysis screen) and my proposal is for a quick demo to be made available. I have a [docker image](https://github.com/8ctopus/apache-php-fpm-alpine) which embeds SPX that could be used for that purpose, however I'm also open to any other option you suggest.

**Comment by NoiseByNorthwest** [comment]
Yes, a Docker-based demo with everything working including the web-UI would be great !
On the other side your Docker image seems to be not focused on SPX which could be confusing for someone coming from a "Try SPX on your PC now" link.

**Comment by 8ctopus** [comment]
The essence of mine is a LAMP sandbox to play / develop with php, so not much more than what's needed to take SPX for a run. In any case, your choice.

**Comment by RichardJECooke** [comment]
I'm struggling to run PHP-SPX even more basically than the big Docker repo in this issue. I'm using a very simple `dockerfile`:

```sh
FROM --platform=linux/amd64 php:8.3
RUN    apt update \
    && apt install zlib1g-dev git -y \
    && cd / \
    && git clone https://github.com/NoiseByNorthwest/php-spx.git \
    && cd php-spx \
    && git checkout release/latest \
    && phpize \
    && ./configure \
    && make \
    && make install
```

with a `app/php.ini` file like this:

```
error_reporting = E_ALL 
display_errors = off
log_errors = on
error_log = /app/log.log

extension=spx.so
zlib.output_compression = 0
spx.http_enabled = 1
spx.http_profiling_enabled = 1
spx.http_key="dev"
spx.http_ip_whitelist="0.0.0.0"
```

And I'm running the PHP built-in web server like so:

```sh
docker build --platform linux/amd64 -t phpspx -f dockerfile .  

docker run --name phpbox --rm -v "./app:/app" -p 8000:8000 --platform linux/amd64 phpspx php -S 0.0.0.0:8000 -t /app
```

But when I browse to http://localhost:8000/?SPX_KEY=dev&SPX_UI_URI=/, I just get the `app/index.php` page. Is there some setting I'm missing please?  Or does PHP-SPX not support the built-in web server?  

I don't know what else to try. Thanks.

**Comment by NoiseByNorthwest** [comment]
Is there something logged in `/app/log.log` ? See also this working example of Web UI + PHP's embedded server https://github.com/NoiseByNorthwest/php-spx/issues/201#issuecomment-1279780817

**Comment by RichardJECooke** [comment]
Got it to work, thanks. Updated your readme here - https://github.com/NoiseByNorthwest/php-spx/pull/257
